### PR TITLE
Jwilaby/babel builds

### PR DESCRIPTION
**Pull request by justinwilaby**

### CODE REVIEW

File: packages/sdk/client/lib/extensions/host.js [hunk 1]
```diff
@@ -0,0 +1 @@
+"use strict";
```
Comment (tonyanziano) [inline packages/sdk/client/lib/extensions/host.js]: newline

File: packages/sdk/client/lib/extensions/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Object.keys(_host).forEach(function (key) {
+  if (key === "default" || key === "__esModule") return;
+  Object.defineProperty(exports, key, {
+    enumerable: true,
+    get: function get() {
+      return _host[key];
+    }
+  });
+});
```
Comment (tonyanziano) [inline packages/sdk/client/lib/extensions/index.js]: newline

File: packages/sdk/client/lib/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Object.keys(_extensions).forEach(function (key) {
+  if (key === "default" || key === "__esModule") return;
+  Object.defineProperty(exports, key, {
+    enumerable: true,
+    get: function get() {
+      return _extensions[key];
+    }
+  });
+});
```
Comment (tonyanziano) [inline packages/sdk/client/lib/index.js]: newline

File: packages/app/client/src/data/reducer/editor.ts [hunk 4]
```diff
@@ -76,7 +76,7 @@ const DEFAULT_STATE: EditorState = {
   docsWithPendingChanges: []
 };
 
-export default function editor(state: EditorState = DEFAULT_STATE, action: EditorAction | BotAction): EditorState {
+export const editor = (state: EditorState = DEFAULT_STATE, action: EditorAction | BotAction): EditorState => {
```
[the remark this answers is not in the document]
Comment (justinwilaby) [reply packages/app/client/src/data/reducer/editor.ts]: This was a special case. For some reason, using `export function` caused a failure and using const worked. We can revisit this once we hammer on babel for a while.

**Pull request merged**